src/BloomExe/BloomExe.csproj

```diff
@@ -1170,11 +1170,13 @@ copy /Y "$(SolutionDir)\lib\dotnet\Interop.Acrobat.dll" $(OutDir)
   </Target>
   -->
   <Target Name="BeforeBuild">
-    <!-- Copy the appropriate version of icudotnet -->
+    <!-- For Linux, copy the appropriate version of icudotnet -->
     <!-- $(icu-config ...) is passed to shell to exec. -->
     <!-- Escape the $ (%24) to ensure it is sent to the shell. -->
     <Exec Command="cp -p $(SolutionDir)/lib/dotnet/icu%24(icu-config --version|tr -d .|cut -c -2)/icu.net.dll* $(SolutionDir)/lib/dotnet" Condition="'$(OS)'!='Windows_NT'" />
     <Exec Command="[ -e $(SolutionDir)/Mercurial ] || ln -s $(SolutionDir)/Mercurial-%24(uname -m)/Mercurial $(SolutionDir)/Mercurial" Condition="'$(OS)'!='Windows_NT'" />
+    <!-- For Linux, compile the various typescript, jade, and less files in BloomBrowserUI. -->
```

Comment:
I would agree, but I was trying to think of how to get it work for package
building, not just building on my machine.  But that could be done in the
overall build script as well I suppose.  I'm not sure how package building
works on jenkins.
--
Steve McConnel

On Fri, Jan 15, 2016 at 2:24 PM, John Hatton <notifications@github.com>
wrote:

> In src/BloomExe/BloomExe.csproj
> <https://github.com/BloomBooks/BloomDesktop/pull/924#discussion_r49900838>
> :
>
> >      <!-- $(icu-config ...) is passed to shell to exec. -->
> >      <!-- Escape the $ (%24) to ensure it is sent to the shell. -->
> >      <Exec Command="cp -p $(SolutionDir)/lib/dotnet/icu%24(icu-config --version|tr -d .|cut -c -2)/icu.net.dll* $(SolutionDir)/lib/dotnet" Condition="'$(OS)'!='Windows_NT'" />
> >      <Exec Command="[ -e $(SolutionDir)/Mercurial ] || ln -s $(SolutionDir)/Mercurial-%24(uname -m)/Mercurial $(SolutionDir)/Mercurial" Condition="'$(OS)'!='Windows_NT'" />
> > +    <!-- For Linux, compile the various typescript, jade, and less files in BloomBrowserUI. -->
>
> this doesn't really belong here, to my mind. Indeed it may be quite slow
> to do an npm install every time you do a build.
>
> Instead, the npm install would seem to belong where I put it, on the TC
> configuration, as build steps. FWIW.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/BloomBooks/BloomDesktop/pull/924/files#r49900838>.
>